interfaces/prompting/requestprompts/requestprompts.go

```diff
@@ -96,15 +80,6 @@ type promptConstraints struct {
 	originalPermissions []string
```

Comment:
should promptConstraints be exported again?

Reply:
I don't think there's any reason for it to be exported? `promptConstraints` (and `Prompt`, for that matter) are only used within the `requestprompts`, aside from when they're marshalled in the future.